install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,673 @@
+ 
+|`user`
+|The user name for authenticating to the registry.
+|N
+
+|`pass`
+|The password for authenticating to the registry. 
+|N
+
+|`org`
+|The namespace or organization that the image is contained in.
+|N
+
+|`type`
+|The type of registry. Available adapters are `mock`, `rhcc`, `openshift`,
+`dockerhub`, and `local_openshift`.
+|Y
+
+|`namespaces`
+|The list of namespaces to configure the `local_openshift` registry type with. By default, a user should use `openshift`.
+|N
+
+|`url`
+|The URL that is used to retrieve image information. Used extensively for RHCC while the `dockerhub` type uses hard-coded URLs.
+|N
+
+|`fail_on_error`
+|Should this registry fail, the bootstrap request if it fails. Will stop the execution of other registries loading.
+|N
+
+|`white_list`
+|The list of regular expressions used to define which image names should be allowed through. Must have a white list to allow APBs to be added to the catalog. The most permissive regular expression that you can use is `.*-apb$` if you would want to retrieve all APBs in a registry. See xref:oab-broker-config-apb-filtering[APB Filtering] for more details.          
+|N
+
+|`black_list`
+|The list of regular expressions used to define which images names should never be allowed through. See xref:oab-broker-config-apb-filtering[APB Filtering] for more details.
+|N
+
+|`images`
+|The list of images to be used with an OpenShift Container Registry.
+|N
+|===
+
+[[oab-broker-config-registry-prod-dev]]
+=== Production or Development
+
+A _production_ broker configuration is designed to be pointed at a trusted
+container distribution registry, such as the Red Hat Container Catalog (RHCC):
+
+[source,yaml]
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: https://registry.access.redhat.com
+    tag: v3.7
+    white_list: 
+      - ".*-apb$"
+  - type: local_openshift
+    name: localregistry
+    namespaces:
+      - openshift
+    white_list: []
+----
+
+However, a _development_ broker configuration is primarily used by developers
+working on the broker. To enable developer settings, set the registry name to
+`dev` and the `dev_broker` field in the `broker` section to `true`:
+
+[source,yaml]
+----
+registry:
+  name: dev
+----
+
+[source,yaml]
+----
+broker:
+  dev_broker: true
+----
+
+[[oab-broker-config-registry-storing-creds]]
+=== Storing Registry Credentials
+
+If `auth_type` is set to `config`, the registry credentials will be read from
+the ConfigMap's `user` and `pass` values. If you want to ensure these
+credentials are not publicly accessible, you can configure a registry to use a
+secret in the broker's namespace for registry.
+
+To do so, set `auth_type` to `secret` and `auth_name` to the name of the secret:
+
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: registry.access.redhat.com
+    auth_type: secret
+    auth_name: asb-auth-secret
+----
+
+The associated secret should have the values `username` and `password` defined.
```

Comment:
@dymurray I got this from https://github.com/openshift/ansible-service-broker/blob/master/docs/config.md#storing-registry-credentials-in-a-secretfile and want to make sure it's accurate to say `username` and `password` here for what to put in the secret, versus `user` and `pass` which is what the registry section uses in the ConfigMap.

Reply:
@AlexanderZagaynov Yes `username` and `password` is correct for this.

Reply:
@dymurray I guess you wanted to mention @adellape